index.html

```diff
@@ -4580,6 +4580,319 @@ <h1>Current Issues</h1>
 
   </section>
 
+  <section class="appendix">
+    <h1>What Does a DID Identify?</h1>
+    <p>
+  The DID Core specification states that a <a>DID</a> identifies the
+  <a>DID subject</a>—and that a <a>DID subject</a> can be anything that can be
+  identified with a URI. However some applications of <a>DIDs</a>, particularly
+  those involving the Semantic Web, may need a more precise definition of how
+  DID identification works. That is the purpose of this Appendix.
+    </p>
+    <h2>What types of resources can a DID identify?</h2>
+    <p>
+  Since a <a>DID</a> is a specific type of URI, the answer to this question is
+  provided by section 1.1 of the URI specification [[!RFC3986]]:
+    </p>
+<blockquote>
+    This specification does not limit the scope of what might be a resource;
+    rather, the term "resource" is used in a general sense for whatever might be
+    identified by a URI.  Familiar examples include an electronic document, an
+    image, a source of information with a consistent purpose (e.g., "today's
+    weather report for Los Angeles"), a service (e.g., an HTTP-to-SMS gateway),
+    and a collection of other resources.  A resource is not necessarily
+    accessible via the Internet; e.g., human beings, corporations, and bound
+    books in a library can also be resources.  Likewise, abstract concepts can
+    be resources, such as the operators and operands of a mathematical equation,
+    the types of a relationship (e.g., "parent" or "employee"), or numeric
+    values (e.g., zero, one, and infinity).
+</blockquote>
+  <p>
+  In other words, it does not matter whether a resource is “on” or “off” the
+  Internet—if it can be identified, it can be assigned a URI, and therefore it
+  can be assigned a <a>DID</a>.
+  </p>
+  <h2>How do you know what a DID identifies?</h2>
+  <p>
+  For any <a>DID</a>, the <a>DID controller</a> determines the <a>DID subject</a>.
+  Unfortunately it is all but impossible to determine the <a>DID subject</a>
```

Comment:
It's not necessarily unfortunate -- especially from a privacy perspective.

```suggestion
  It is not expected to be possible to determine the <a>DID subject</a>
```